core-services/release-controller/_releases/release-ocp-4.8-ci.json

```diff
@@ -46,7 +46,7 @@
     },
     "upgrade":{
       "upgrade":true,
-      "optional":true,
+      "optional":false,
```

Comment:
nit: `"optional":false,` doesn't seem to be a common pattern.  Should we just drop this line?